<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditPurgeDlg</class>
 <widget class="QDialog" name="AuditPurgeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audit &amp; purge</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cbPurgeRegApps">
       <property name="text">
        <string>purge regapps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPurgeAll">
       <property name="text">
        <string>purge all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbExplodeProxy">
       <property name="text">
        <string>explode all proxies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbRemoveProxy">
       <property name="text">
        <string>remove all proxies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAudit">
       <property name="text">
        <string>audit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>Accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPurgeRegApps</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbPurgeAll</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbExplodeProxy</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRemoveProxy</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAudit</sender>
   <signal>stateChanged(int)</signal>
   <receiver>AuditPurgeDlg</receiver>
   <slot>CheckAnySelected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Accept()</slot>
  <slot>CheckAnySelected(int)</slot>
 </slots>
</ui>
